clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -188,6 +188,28 @@ class GCEUtil {
     }
   }
 
+  static boolean isShieldedVmCompatible(Image image) {
+    java.util.List<GuestOsFeature> guestOsFeatureList = image.getGuestOsFeatures()
+    if (guestOsFeatureList == null || guestOsFeatureList.size() == 0) {
+      return false
+    }
+
+    boolean isUefiCompatible = false;
+    boolean isSecureBootCompatible = false;
+    for (def feature : guestOsFeatureList) {
```

Comment:
Can do a groovy `.each` here: `guestOsFeatureList.each { feature -> <logic> }`.

Reply:
Not that big of a deal though, pretty readable as-is.

Reply:
Will change it. I am all for using groovy's built-in controls :)